jupyter/src/incremental_learning/config.py

```diff
@@ -12,21 +12,33 @@
 import platform
 from sys import exit
 
-from pathlib2 import Path
+from pathlib import Path
 
-root_dir = Path(__file__).parent.parent.parent.absolute()
+# Check for resource called in ancestor directories of path.
+def find_ancestor_dir(path: Path, resource: str):
```

Comment:
It would be nice to add a bit more explanation here in form of comments or docstrings. I don't fully understand what is happening here.

Reply:
I added fuller documentation in [d4bdd1b](https://github.com/elastic/ml-cpp/pull/1994/commits/d4bdd1bf982a28328c1b8ba5d37b706226ec8ec5).